doc/user/rbac.md

```diff
@@ -0,0 +1,168 @@
+# Operator RBAC setup
+
+If RBAC is in place, users need to setup RBAC rules for etcd operator. This doc serves a tutorial for it.
+
+## Quick setup
+
+If you just want to play with etcd operator and your kubectl have admin access, there is a quick setup.
+
+First, create a new namespace `test-etcd-operator`:
+
+```bash
+$ kubectl create ns test-etcd-operator
+```
+
+Then create cluster role binding:
+
+```bash
+$ cat <<EOF | kubectl create -f -
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: test-etcd-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
```

Comment:
`admin` is a tectonic specific roll. The upstream equivalent is `cluster-admin`. Maybe use that here?